modules/output/equity_plot.py

```diff
@@ -1,29 +1,39 @@
-from datetime import datetime
-from math import log10
 from pathlib import Path
+
 import pandas as pd
+from plotly.graph_objs import Figure
 
-from plotly import graph_objects as go
+from modules.public.trading_stats import TradingStats
+from modules.output.plots import convert_df_for_plotting, compute_average_market_change
 
 
-def equity_plot(capital_dict, strategy_name):
+def equity_plot(stats: TradingStats, strategy_name):
     Path("data/backtesting-data/plots/equity").mkdir(parents=True, exist_ok=True)
-    df = pd.DataFrame(list(capital_dict.items()))
 
-    # remove first value since its 0
-    df.drop(index=df.index[0],
-            axis=0,
-            inplace=True)
+    df_capital = pd.DataFrame(list(stats.capital_per_timestamp.items()), columns=['timestamp', 'capital'])
+
+    df_capital, dates = convert_df_for_plotting(df_capital)
 
-    # get dates and y range
-    dates = [datetime.fromtimestamp(time / 1000) for time in df[0]]
-    min_value = max(int(df[1].min() - 10), 1)
-    max_value = int(df[1].max() + 10)
+    df_coins = compute_average_market_change(stats.df, stats.capital_per_timestamp[0])
```

Comment:
Be mindful of your variables - while it's clear to you that df_coins means the average market change over all the coins normalized by the starting capital, that might not be clear to other people reading your code.

How about 'avg_market_change_coins'? (Be sure to change it elsewhere in the code too

```suggestion
    avg_market_change_coins = compute_average_market_change(stats.df, stats.capital_per_timestamp[0])
```